README.md

```diff
@@ -1 +1,39 @@
-# hello-microverse-project
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
```

Comment:
- [ ] The Branch should have a name of a feature or action (e.g. 'created_sidebar' or 'added_comments_section'), kindly keep this in mind for future projects